CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 This file is used to list changes made in each version of grafana.
 
+### 4.0.3 (2019-04-25)
```

Comment:
Set to unreleased, as version number has not been decided :)

Reply:
@Xorima I have set it to unreleased :)